fan-xiaomi.js

```diff
@@ -66,19 +86,24 @@ class FanXiaomi extends HTMLElement {
                     let icon = u.querySelector('.icon-waper > iron-icon')
                     let newSpeed
                     if (icon.getAttribute('icon') == "mdi:numeric-1-box-outline") {
-                        newSpeed = 40
+                        newSpeed = speed_list['Level 2']
```

Comment:
```suggestion
                        newSpeed = speed_list[1]
```
Arrays here in the speed selection